view/theme/vier/config.php

```diff
@@ -9,6 +9,9 @@ function theme_content(&$a){
 	if(!local_user())
 		return;
 
+	if (!function_exists('get_vier_config'))
+		return;
```

Comment:
could such a check made automatically from the core, so theme designers don't have to worry about it?